### ISSUE TITLE
Design update to heading and title styles of public profile pages

### ISSUE THREAD

**Issue by Memo-Es** [opencollective/opencollective#5112]
## Context

The text styles of the titles, subtitles, and headings of public profile pages are somehow off, for example:

| Description | Version in production | Design proposal |
|-----------------------|-----------------|-----------------|
|   Here, the collective's name (Also the page title) is competing with the next title, should be subordinated to the main page title. Right now, bot are 32 px, we are changing the collective's name to 40 px    |  ![Annotation on 2022-01-10 at 21-33-38](https://user-images.githubusercontent.com/49539954/148877904-0cc881e5-a312-487c-8232-c7e1a2231ea3.png)  |   <img width="1981" alt="Screen Shot on 2022-01-10 at 21-51-14" src="https://user-images.githubusercontent.com/49539954/148878356-ae7a47f4-afbe-4b99-98d8-c906fe4febe7.png">   |
| Similar to the previous one, we have a subtitle with the same style as the title. We changed it from 32 px both to 32 px and 24 px respectively |   ![Annotation on 2022-01-10 at 21-37-37](https://user-images.githubusercontent.com/49539954/148877910-ba07a075-14bb-48fe-84fe-497136bb5eeb.png)   | <img width="1273" alt="Screen Shot on 2022-01-10 at 21-59-00" src="https://user-images.githubusercontent.com/49539954/148879245-a30d00fd-c99e-4f45-94a9-2720e33c6675.png">  |
| In this case, we have yet another title style, this one even bigger than the page title. We changed it from 40 px to use a subtitle in 24 px |   ![Annotation on 2022-01-10 at 21-38-30](https://user-images.githubusercontent.com/49539954/148877912-2f664c68-2119-4cdd-b47a-e8a2d0d45a70.png)   |   <img width="1277" alt="Screen Shot on 2022-01-10 at 22-07-18" src="https://user-images.githubusercontent.com/49539954/148879616-12e86bd3-467d-4565-8ac9-6632fc444207.png">      |

We will have lesser, more consistent styling in headings and titles with these updates.

🎨 [Design file reference](https://www.figma.com/file/e71tBo0Sr8J7R5n6iMkqI42d/%5BOC.com%5D-04-Collectives?node-id=11750%3A5968)

cc @Betree

**Comment by Betree** [comment]
@Memo-Es Is the design linked above ready for implementation?

**Comment by Memo-Es** [comment]
@Betree Yes, literally is just having a descending order in the text styles from 40 - 32 - 24 px so I made the changes directly in the library file, that's why is not in production ready

**Comment by Betree** [comment]
This was done
